geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/analytics/HistogramJob.scala

```diff
@@ -151,11 +166,13 @@ object HistogramJob {
              feature: String,
              attribute: String,
              groupBy: List[String] = List.empty,
-             uniqueBy: List[String] = List.empty) = {
+             uniqueBy: List[String] = List.empty,
+             writeToAccumulo: List[String] = List.empty) = {
```

Comment:
can make this a boolean and just put it into a list for the map below (scalding requires everything to be a list, but this interface doesn't have to reflect that)